schema/rpc.fbs

```diff
@@ -338,20 +344,71 @@ table AutoBoneProcessRequest {
 
 table AutoBoneProcessStatusResponse {
     process_type: AutoBoneProcessType;
-    message: string;
+    /// The current count. This value is -1 if there is nothing to report.
     current: uint32;
+    /// The total count. This value is -1 if there is nothing to report.
     total: uint32;
+    /// The time remaining in seconds. This value is -1 if there is nothing to report.
+    eta: float32;
+    /// True if the operation has completed with any result, successful or not.
     completed: bool;
+    /// True if the completed operation was successful, only observe if `completed` is true.
     success: bool;
 }
 
 table AutoBoneEpochResponse {
     current_epoch: uint32;
     total_epochs: uint32;
+    /// The current error value. This can be any positive number, where lower is better.
     epoch_error: float32;
+    /// A list of the current estimated body proportions.
     adjusted_skeleton_parts: [SkeletonPart];
 }
 
+table AutoBoneSettings {
```

Comment:
At least give a permalink to where these fields are in the Java code, if you don't want to comment them all

Reply:
This needs to be a permalink, not a link to the latest main